stream-chat-android-client/src/main/java/io/getstream/chat/android/client/uploader/StreamFileUploader.kt

```diff
@@ -41,17 +41,15 @@ internal class StreamFileUploader(
         connectionId: String,
         file: File,
     ): Result<String> {
-        val part = MultipartBody.Part.createFormData(
-            "file",
-            file.name,
-            file.asRequestBody(file.getMediaType())
-        )
+        val body = file.asRequestBody(file.getMediaType())
+        val part = MultipartBody.Part.createFormData("file", file.name, body)
 
         return retrofitCdnApi.sendFile(
-            channelType,
-            channelId,
-            part,
-            connectionId
+            channelType = channelType,
+            channelId = channelId,
+            file = part,
+            connectionId = connectionId,
+            progressCallback = null,
```

Comment:
It would be nice to have just a single `sendFile` method here that takes a nullable `ProgressCallback` as a parameter that is `null` by default. However, this would be a breaking change for our clients that have existing implementations of `StreamFileUploader`.